frontend/app/src/App.tsx

```diff
@@ -46,7 +46,7 @@ import {
   ForwardMsgMetadata,
   generateUID,
   getCachedTheme,
-  getElementWidgetID,
+  getElementID,
```

Comment:
nit: should we use the opportunity and call this `getElementId`? While I don't thing we have codified our style consistently, we should perhaps just use an existing style guide and Google's style guide for JavaScript would use the suggested variant [here](https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined).

Reply:
Sounds good, updated it 👍